app/views/AccountHomeView.scala.html

```diff
@@ -33,7 +33,7 @@
 
 @theRows = @{for(app <- applications) yield Seq(
                     TableRow(content = Text(app.id), format = Some("text")),
-                    TableRow(content = Text(app.goodsName), format = Some("text")),
+                    TableRow(content = Text(if (app.goodsName.trim.isEmpty) messages("accountHome.noDescription") else app.goodsName), format = Some("text")),
```

Comment:
I'm a little concerned that this has nothing testing it hasn't broken. We could either move it to a viewmodel (but that seems OTT for this). Can you add a test for this page to check for the accountHome.noDescription appearing in the HTML in the AccountHomeControllerSpec